TouchEffect/TouchEffect.csproj

```diff
@@ -8,9 +8,6 @@
     <PackageReference Include="Xamarin.Forms" Version="4.2.0.848062" />
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="Delegates\" />
-    <Folder Include="EventArgs\" />
-    <Folder Include="Enums\" />
```

Comment:
why removing them?

Reply:
Just a mistake, thanks for pointing it out